pkg/clusteragent/admission/util.go

```diff
@@ -0,0 +1,114 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+// +build kubeapiserver
+
+package admission
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/util/kubernetes/apiserver/common"
+
+	admiv1beta1 "k8s.io/api/admissionregistration/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	admissionEnabledLabelKey = "admission.datadoghq.com/enabled"
+	envTagLabelKey           = "tags.datadoghq.com/env"
+	serviceTagLabelKey       = "tags.datadoghq.com/service"
+	versionTagLabelKey       = "tags.datadoghq.com/version"
```

Comment:
do we have this tags in another place already?

Reply:
yes in the tagger under `+build kubelet` build tag which is not included in the cluster agent binary, I will move them to a common package so we can import them